Makefile

```diff
@@ -18,6 +18,13 @@
 # set the shell to bash in case some environments use sh
 SHELL:=/bin/bash
 
+# Default behaviour is to use buildx only for github actions, until the Travis workflow is deprecated.
+BUILDX:=false
```

Comment:
Do we have a timeline when this will occur?

Reply:
For this repo, to deprecate travis, the following tasks should be completed:
1. Include the tests into github actions
2. This repo is the starting point of [release tagging](https://github.com/openebs/openebs/blob/master/contribute/process/release-management.md#release-tagging) and all other repos are automatically tagged once the travis build from `linux-utils` passes. So we need to migrate the tagging script also to github actions.

I think after the above 2 steps, travis can be deprecated. WDYT: @kmova